config/data/langs.toml

```diff
@@ -22,6 +22,24 @@ each argument is NULL terminated.
 x is a no-op.
 '''
 
+[APL]
+args       = [ '/usr/bin/apl', '-' ]
+experiment = 135
+size       = '14.5 MiB'
+version    = 'Dyalog APL 19.0.48958'
+website    = 'https://www.dyalog.com'
+example    = '''
+⍝ Printing
+⎕←'Hello, World!'
+
+⍝ Looping
+⎕IO←0
+⎕←¨⍳10
```

Comment:
Can we write `{⎕←⍵}¨⍳10` instead? `⎕←¨` is not supported syntax.

Reply:
@abrudz

For sure, we could do that. The current sample produces perfectly fine output, though.

![image](https://github.com/user-attachments/assets/7e17f0e9-46c3-4f4b-93b9-1058833720a0)

What's your git commit e-mail address? Eventually, if this gets merged and you didn't hide it, a new achievement's coming your way.

Reply:
Why do you need my git commit e-mail address? (I use two, private and work, but interchangably, so…)

Reply:
> Why do you need my git commit e-mail address? (I use two, private and work, but interchangably, so…)

<img width="74" alt="pair-extraordinaire-default-579438a20e01" src="https://github.com/user-attachments/assets/61a40092-f35e-4054-a5fe-a93a9a5ca02c">

But never mind then, s'all good man.